rfcs/progress-steps-component.md

```diff
@@ -0,0 +1,88 @@
+# Progress Steps Component
+
+Progress steps are used for multi-step flows where indicating the number of steps in a process is important.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {ProgressSteps} from 'baseui/progress-steps';
+
+export default () => {
+  return (
+    <ProgressSteps current={2}>
+      <Step title="Step 1" />
+      <Step title="Step 2" />
+      <Step title="Step 3" />
+    </ProgressSteps>
+  );
+}
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {ProgressSteps, StyledNumberedStep} from 'baseui/progress-steps';
+
+export default () => {
+  const overrides ={
+    Step: props => <StyledNumberedStep {...props}/>,
+    StepTitle: props => <CustomTitle {...props} />,
+  };
+  return
+    <ProgressSteps
+      onClick={()=>{console.log('step is clicked')}}
```

Comment:
will onclick be called with the step index?

Reply:
yes! updated